<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>行人检测追踪</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0">
    <item row="0" column="0" rowspan="8" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>828</width>
         <height>550</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_add_video">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>680</width>
            <height>500</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>90</pointsize>
           </font>
          </property>
          <property name="statusTip">
           <string>点击添加视频源</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(59, 59, 59);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QFrame" name="frame_detect">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="6" column="0" colspan="3">
        <widget class="QRadioButton" name="radioButton_parallel">
         <property name="text">
          <string>CPU多线程</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_confidence">
         <property name="text">
          <string>置信度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>检测</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QComboBox" name="comboBox_model">
         <item>
          <property name="text">
           <string>单帧检测模型</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>时间依赖检测模型</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="5">
        <widget class="QPushButton" name="btn_process_detect">
         <property name="text">
          <string>开始检测</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QSlider" name="slider_confidence">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QRadioButton" name="radioButton_use_gpu">
         <property name="text">
          <string>GPU加速</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="2">
        <widget class="QSpinBox" name="spinBox_parallel">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_model">
         <property name="text">
          <string>模型</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="5">
        <widget class="QCheckBox" name="checkBox_preview_detect">
         <property name="text">
          <string>显示处理预览</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QFrame" name="frame_track">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_process_track">
         <property name="text">
          <string>开始追踪</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>追踪</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_preview_track">
         <property name="text">
          <string>显示处理预览</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_process">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>等待处理</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>209</width>
        <height>38</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>209</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="3" rowspan="2">
     <widget class="QFrame" name="frame_select">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_select_win">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>选择目标</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_result">
         <property name="text">
          <string>查看结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="btn_stop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QProgressBar" name="progressBar_process">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_add_video"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_add_video">
   <property name="text">
    <string>添加源</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
